packages/ui-core/src/components/slider/control/control.component.tsx

```diff
@@ -27,6 +29,8 @@ export const Control: FC<Props> = ({
   const styles = {
     display: 'inline-block',
     position: 'absolute',
+    zIndex: 1,
```

Comment:
Is that really required here ? The `controlStyles` type contain the `x` any `y` property. For reference please check `RangeSlider` component.

```
controlStyles={{ [isHorizontal ? 'x' : 'y']: dimension }}
```

Reply:
there is an issue with elements order, the tooltip is covered by axis label.

please take a look:
- w/o `zIndex`

<img width="566" alt="Screenshot 2020-04-27 at 12 48 18" src="https://user-images.githubusercontent.com/23423731/80364053-857fe200-8885-11ea-9056-af42165c4d44.png">

- with `zIndex`
<img width="565" alt="Screenshot 2020-04-27 at 12 48 43" src="https://user-images.githubusercontent.com/23423731/80364057-86b10f00-8885-11ea-8b86-f65ec9411d6f.png">